config/default.json

```diff
@@ -43,6 +43,7 @@
     "features": "blf-features",
     "session": "blf-alpha-session"
   },
+  "extendedCookieDurationInDays": 7,
```

Comment:
This feels more like a constant in that it's across all environments + specific to one controller so I'd be inclined to inline it.